peer/ping_manager.go

```diff
@@ -231,6 +237,9 @@ func (m *PingManager) pingHandler() {
 			m.pingTime.Store(&actualRTT)
 			m.resetPingState()
 
+		case <-m.cfg.PeerCtx.Done():
+			return
+
 		case <-m.quit:
 			return
```

Comment:
we should not need to do this:

the `m.quit` channel is already closed if the PingManager's `Stop` method is called. And today it is already called in ` (p *Brontide) Disconnect`. 

And this is done just before the Brontide's ContextGuard's `Quit` method is called - which would cause this `PeerCtx` to be cancelled anyways. 

So this commit is not needed

Reply:
Dropped the last commit all together.